backend/src/config/config.schema.ts

```diff
@@ -39,143 +39,143 @@ export interface ConfigSchema {
     host: string
     port: number
   }
-  health: { heapSizeThreshold: number; rssThreshold: number }
+  health: { heapSizeThreshold: number, rssThreshold: number }
 }
 
 addFormats({
   'required-string': {
     validate: (val?: string): void => {
-      if (val == undefined || val === '') {
+      if (val === undefined || val === null || val === '') {
```

Comment:
```suggestion
      if (!val) {
```

Would this be equivalent?

Reply:
`strict-boolean-expressions`, unfortch